Build failed in Apple's M1 chip

**Describe the bug**
Build failed on Apple's M1 chips

Failed reason:
```
Caused by: java.lang.Exception: No native library is found for os.name=Mac and os.arch=aarch64. path=/org/sqlite/native/Mac/aarch64 in android.
```

**To Reproduce**
Steps to reproduce the behavior:

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Smartphone (please complete the following information):**
 - Device: [e.g. Google Pixel 2]
 - OS: [e.g. Android 9]
 - Rooted: [e.g. No]
 - Version: [e.g. 3.3.4]

**Additional context**
Add any other context about the problem here.

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -6,7 +6,7 @@ apply plugin: 'com.hiya.jacoco-android'
 apply plugin: "com.starter.easylauncher"
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
 
     packagingOptions {
         exclude 'proguard-project.txt'

--- a/build.gradle
+++ b/build.gradle
@@ -8,7 +8,7 @@ buildscript {
         sshjVersion = '0.32.0'
         jcifsVersion = '2.1.6'
         fabSpeedDialVersion = '3.1.1'
-        roomVersion = '2.3.0'
+        roomVersion = '2.4.0-rc01'
         bouncyCastleVersion = '1.68'
         awaitilityVersion = "3.1.6"
         androidMaterialVersion = "1.4.0"

--- a/build.gradle
+++ b/build.gradle
@@ -8,7 +8,7 @@ buildscript {
         sshjVersion = '0.32.0'
         jcifsVersion = '2.1.6'
         fabSpeedDialVersion = '3.1.1'
-        roomVersion = '2.4.0-rc01'
+        roomVersion = '2.4.0'
         bouncyCastleVersion = '1.68'
         awaitilityVersion = "3.1.6"
         androidMaterialVersion = "1.4.0"

